source/config/configSpec.py

```diff
@@ -258,6 +258,9 @@
 [featureFlag]
 	# 0:default, 1:yes, 2:no
 	cancelExpiredFocusSpeech = integer(0, 2, default=0)
+
+[iaccessible]

```

Comment:
```suggestion
[IAccessible]

```

Reply:
Since there's now a dedicated config section for feature flags, do we even need to make a new section at all?

Reply:
Whether to make a new section really depends on the future intent for the setting. If the setting will still be configurable when it is no longer experimental, then it should go straight to it's appropriate category. If the intent is to eventually remove the setting entirely, then it should live in the feature flags section until that is true.